.buildkite/pipeline.yml

```diff
@@ -44,7 +44,7 @@ steps:
   - label: "cargo test nightly not integration-tests*"
     command: |
       source ~/.cargo/env && set -eux
-      cargo test --workspace --features nightly,test_features -p '*' --exclude 'integration-tests*'
+      cargo test --workspace --features nightly,test_features,mock_node -p '*' --exclude 'integration-tests*'
```

Comment:
if we have the `mock_node` feature for all the tests, then will we skip this check here?

https://github.com/near/nearcore/blob/2de97f4032484c6ca3bbf8697cfdd5f11a229d07/chain/chain/src/chain.rs#L3560-L3572

not sure if any tests depend on that directly but it feels kind of sad to have to skip that

Reply:
yea makes sense. and on the last bullet point, maybe better yet is to make mock-node somehow handle that case and not need any differences in the that check :)

Reply:
Yeah, the last one would be perfect! I didn't look into that tbh, just wanted to come up with a minimal diff to unbreak tests :)